azure/iac/master-compliance-test.json

```diff
@@ -313,8 +313,8 @@
                             "id": "PR-AZR-0126-ARM",
                             "eval": "data.rule.storage_blob_soft_delete",
                             "message": "data.rule.storage_blob_soft_delete_err",
-                            "remediationDescription": "",
-                            "remediationFunction": ""
+                            "remediationDescription": "In Resource of type \"Microsoft.storage/storageaccounts/blobservices\" make sure properties.deleteRetentionPolicy.enabled exists and value is set to true . \n Please visit https://docs.microsoft.com/en-us/azure/templates/microsoft.storage/storageaccounts/blobservices for details.",
```

Comment:
@shahinshirvani  `For resource type ...`

Reply:
fixed